src/connection_manager.c

```diff
@@ -240,12 +247,15 @@ static qd_error_t load_server_config(qd_dispatch_t *qd, qd_server_config_t *conf
     config->allowInsecureAuthentication = true;
     config->verify_host_name = verifyHostName;
 
+    char *stripAnnotations  = qd_entity_opt_string(entity, "stripAnnotations", 0);
```

Comment:
yes, the change here is to group it closer together with where it is being used and to make sure to free it before the CHECK() call happens.  Otherwise, CHECK might cause the method to be exited before the memory is freed.